### PR TITLE
Query to get problematic payments

### PR DESCRIPTION
**Pull request by markiyanshvets**

### CODE REVIEW

File: Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs [hunk 1]
```diff
@@ -14,6 +14,18 @@ namespace BccPay.Core.Sample.API.Controllers
     [Route("[controller]")]
     public class PaymentController : BaseController
     {
+        [HttpGet("problematic")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(ProblematicPaymentsResult))]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> GetProblematicPayments()
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs:20]: [Authorize(Roles = "Administrator")]

**Review by yuriy-butkevych — APPROVED** [review]

**Comment by markiyanshvets** [comment]
image

File: Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs [hunk 2]
```diff
@@ -14,8 +14,8 @@ namespace BccPay.Core.Sample.API.Controllers
     [Route("[controller]")]
     public class PaymentController : BaseController
     {
-        [HttpGet("problematic")]
-        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(PaymentsResults))]
+        [HttpGet("filters")]
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs:17]: Since it is not `filters`, could you rename it to `filtered`?

**Review by yuriy-butkevych — APPROVED** [review]

File: Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs (hunk 2 above)
Comment (markiyanshvets) [reply Api/BccPay.Core.Sample/BccPay.Core.Sample.API/Controllers/PaymentController.cs]: Okay

**Pull request merged**